pages/exercises/[lessonSlug].tsx

```diff
@@ -54,10 +54,10 @@ const Exercises: React.FC<QueryDataProps<GetExercisesQuery>> = ({
   const currentExercises = exercises
     .filter(exercise => exercise?.module.lesson.slug === slug)
     .map(exercise => ({
-      challengeName: exercise?.module.name!,
-      problem: exercise?.description!,
-      answer: exercise?.answer!,
-      explanation: exercise?.explanation!
+      challengeName: exercise.module.name,
+      problem: exercise.description,
+      answer: exercise.answer,
+      explanation: exercise.explanation || ''
```

Comment:
[Exercise explanations are nullable](https://github.com/garageScript/c0d3-app/blob/master/prisma/schema.prisma#L187) so we have to handle it here by defauting to an empty string, which works fine and looks good (see pull request description for a picture of what it looks like).